src/System.Memory/tests/Span/SequenceCompareTo.bool.cs

```diff
@@ -0,0 +1,149 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static partial class SpanTests
+    {
+        [Fact]
+        public static void ZeroLengthSequenceCompareTo_Bool()
+        {
+            var a = new bool[3];
+
+            var first = new Span<bool>(a, 1, 0);
+            var second = new Span<bool>(a, 2, 0);
```

Comment:
nit: If you will notice, the second Span gets implicitly casted to a ReadOnlySpan when we call SequenceCompareTo. Consider changing the type to ReadOnlySpan explicitly:
```C#
var second = new ReadOnlySpan<bool>(a, 2, 0);
```

This way we are consistent with other tests, like SequenceCompareToWithSingleMismatch_Bool https://github.com/dotnet/corefx/pull/28320/files#diff-b162e342a6ea095c71071f76c985a098R91